edgecast/resources/cps/certificate.go

```diff
@@ -422,8 +422,53 @@ func ResourceCertificateRead(ctx context.Context,
 	d *schema.ResourceData,
 	m interface{},
 ) diag.Diagnostics {
-	// Not Yet Implemented
 	var diags diag.Diagnostics
+
+	config := m.(**api.ClientConfig)
+	cpsService, err := buildCPSService(**config)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	params := certificate.NewCertificateGetParams()
+	n, err := strconv.ParseInt(d.Get("id").(string), 10, 64)
+	if err == nil {
+		fmt.Printf("%d of type %T", n, n)
+	}
+	params.ID = n
+	resp, err := cpsService.Certificate.CertificateGet(params)
+	if err != nil {
+		d.SetId("")
```

Comment:
I don't believe we should set the Id == "" here as any API error would indicate to Terraform the ID should be unset (since this is a read and presumably the resource already exists and the ID is valid).